server/src/util/sh.ts

```diff
@@ -0,0 +1,25 @@
+import * as ChildProcess from 'child_process'
+
+/**
+ * Execute the following sh program.
```

Comment:
Nitpicking: I prefer not having code comments like this and make the function name more descriptive: `execShellScript(body: string)`

Reply:
We could use https://github.com/sindresorhus/execa for all child process handling. It has a few advantages but also introduces more dependencies. Pros cons.

Reply:
@skovhus Let's switch to `execa` if we find bugs in this code :)